interface/auth.py

```diff
@@ -125,8 +125,8 @@ def load_public_from_str(key: str):
 
     def to_json_key(self) -> str:
         """new line characters are formatted to \\n"""
```

Comment:
Considering that the functionality of the function is being changed to provide just the `public_key()` without any formatting, maybe the definition ought to be altered as well?

Reply:
It was recommended somewhere that PEM be formatted with newline characters replaced with `\n`. An earlier implementation used ASCII instead of utf-8 to encode/decode keys, which required special string replacement(see commented out line).

For the time being, I'd like to retain this method as-is because we might run into issues when testing against AP instances but [I've taken note of this](https://github.com/forgeflux-org/interface/projects/1#card-76140146) we'll comeback to this during cleanup.

Most of this AP stuff is being implemented on a hunch(as there are no test suites), we won't know what works and what doesn't until we get the write-end of the pipeline working.

Reply:
Alright, that does seem like a logical route to follow through with.
No issues found otherwise, this PR is good to go for a merge.